docs/griptape-framework/structures/tasks.md

```diff
@@ -146,3 +146,254 @@ agent.run()
                              Output: The summary of the webpage https://www.griptape.ai has been successfully stored in a file named
                              griptape.txt.
 ```
+
+
+## Extraction Task
+
+To extract information from a text, you can use the ExtractionTask. This task allows us to pass in an Extraction Engine and a set of arguments to the engine.
+
+```python
```

Comment:
Create two distinct sections under Extraction Task for JSON Extraction Task and CSV Extraction Task. Each section should contain its own example.

Reply:
ok!

Reply:
Let's add section headers for each of the extraction tasks. `### JSON Extraction` and `CSV Extraction`. And we should probably explain the `column_names` and `template_schema` in each section.